### ISSUE TITLE
Exception: builtins.AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#168]
```
2017-08-08T14:31:03-0700 [ims.application._klein#critical] Request failed
        Traceback (most recent call last):
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 332, in addErrback
            errbackKeywords=kw)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 310, in addCallbacks
            self._runCallbacks()
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
            current.result = callback(current.result, *args, **kw)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/resource.py", line 260, in processing_failed
            failure)
        --- <exception caught here> ---
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
            result = f(*args, **kw)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 133, in execute_error_handler
            return handler(self._instance, request, failure)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 323, in _f
            r = _call(instance, f, request, failure)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 48, in _call
            result = f(instance, *args, **kwargs)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/application/_klein.py", line 69, in wrapper
            response = f(self, request, *args, **kwargs)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/application/_klein.py", line 325, in requestRedirectError
            url = URL.fromText(failure.value.args[0].decode("utf-8"))
        builtins.AttributeError: 'str' object has no attribute 'decode'
```

**Comment by wsanchez** [comment]
Should be fixed in #175

**Comment by wsanchez** [comment]
Merged
